runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/AbstractDokkaTask.kt

```diff
@@ -36,6 +36,10 @@ abstract class AbstractDokkaTask(
     val failOnWarning: Property<Boolean> = project.objects.safeProperty<Boolean>()
         .safeConvention(DokkaDefaults.failOnWarning)
 
+    @Input
+    val globalMainPage: Property<String> = project.objects.safeProperty<String>()
```

Comment:
Hmm, maybe we should set this up: We should know this information and I would wait until somebody comes up with a good use case for making this configurable.